src/src/main/java/dao/jdbc/BasketJDBC.java

```diff
@@ -17,7 +17,7 @@
 
 public class BasketJDBC implements BasketDAO {
     private static final Logger logger = Logger.getLogger(ProductJDBC.class);
-    private static final JDBCService jdbcService = new JDBCService();
+    private static final JDBCUtil JDBC_UTIL = new JDBCUtil();
```

Comment:
we don't need to create object of this class, we can write in method: 
Connection connection = JDBCUtil.getConnection()